communication/src/main/java/PNCInterface.java

```diff
@@ -0,0 +1,31 @@
+import org.jboss.da.communcation.pnc.entity.BuildConfiguration;
+import org.jboss.da.communcation.pnc.entity.BuildConfigurationSet;
+import org.jboss.da.communcation.pnc.entity.Product;
+import org.jboss.da.communcation.pnc.entity.Project;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import java.util.List;
+
+public interface PNCInterface {
+
+    @GET
+    @Path("pnc-rest/rest/build-configurations")
```

Comment:
So, the target will become "<pnc-website>/pnc-rest/rest"?

I don't know. I kinda like that I just have to specify the main url in the target and not have to remember to add the `/pnc-rest/rest`. What does @jbartece or @slaskawi think?